src/devices/ovrheadset/OVRHeadset.cpp

```diff
@@ -61,6 +63,8 @@ static constexpr unsigned int BUTTON_COUNT = 13;
 #undef check
 #endif
 
```

Comment:
@barbalberto can you please tell us if this comment is still valid or if it refers to an outdated commit?